src/containers/SideMenu.js

```diff
@@ -155,7 +155,7 @@ const styles = StyleSheet.create({
   sceneContainer: {
     flex: 1,
     marginLeft: -15,
-    flexDirection: 'row'
```

Comment:
@jrvanderveen looks sweet! I think this is the only line that should be reverted back to existence and then we're ready to merge!